test/unit/renderer/viz/expressions/buckets.test.js

```diff
@@ -209,31 +210,31 @@ describe('src/renderer/viz/expressions/buckets', () => {
                 });
 
                 it('should classify the input in the second category when is equal than the first breakpoint', () => {
-                    const expected = 1;
+                    const expected = 0.5;
                     const feature = { price: 10 };
                     const actual = bucketExpression.eval(feature);
 
                     expect(actual).toEqual(expected);
                 });
 
                 it('should classify the input in the second category when is lower than the second breakpoint', () => {
-                    const expected = 1;
+                    const expected = 0.5;
                     const feature = { price: 15 };
                     const actual = bucketExpression.eval(feature);
 
                     expect(actual).toEqual(expected);
                 });
 
                 it('should classify the input in the third category when is equal than the third breakpoint', () => {
-                    const expected = 2;
+                    const expected = OTHERS_INDEX;
```

Comment:
No, right? there is no `others` when using buckets with a numeric property

Reply:
The `eval` method in buckets does this check for numeric properties:  `if (v < this.list.elems[i].eval(feature))`, otherwise returns -1, so the last bucket is considered as 'others'. We may should change the eval in buckets:

* current code:
```js
const v = this.input.eval(feature);
const divisor = this.numCategoriesWithoutOthers - 1 || 1;

if (this.input.type === 'category') {
    for (let i = 0; i < this.list.elems.length; i++) {
        if (v === this.list.elems[i].eval(feature)) {
            return i / divisor;
        }
    }
} else if (this.input.type === 'number') {
    for (let i = 0; i < this.list.elems.length; i++) {
        if (v < this.list.elems[i].eval(feature)) {
            return i / divisor;
        }
    }
}

return OTHERS_INDEX;
```

* proposal:
```js
const v = this.input.eval(feature);
const divisor = this.numCategoriesWithoutOthers - 1 || 1;

if (this.input.type === 'category') {
    for (let i = 0; i < this.list.elems.length; i++) {
        if (v === this.list.elems[i].eval(feature)) {
            return i / divisor;
        }
      
      return OTHERS_INDEX;
    }
}

for (let i = 0; i < this.list.elems.length; i++) {
    if (v < this.list.elems[i].eval(feature)) {
        return i / divisor;
    }
}

return 1;
```

What do you think?